tls/s2n_kem.c

```diff
@@ -237,6 +237,10 @@ int s2n_kem_free(struct s2n_kem_params *kem_params)
         if (kem_params->private_key.allocated) {
             GUARD(s2n_free(&kem_params->private_key));
         }
+        GUARD(s2n_blob_zero(&kem_params->shared_secret));
+        if (kem_params->shared_secret.allocated) {
+            GUARD(s2n_free(&kem_params->shared_secret));
+        }
```

Comment:
This pattern is repeated several times, can you make it into a function? Maybe `s2n_blob_zeroize_free()`

Then you just need to call
```
s2n_blob_zeroize_free(&kem_params->private_key);
s2n_blob_zeroize_free(&kem_params->shared_secret);
s2n_blob_zeroize_free(&kem_params->public_key);
```

Reply:
Good call.